cfme/tests/v2v/test_v2v_manual.py

```diff
@@ -0,0 +1,632 @@
+"""Manual tests"""
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [test_requirements.v2v, pytest.mark.manual]
+
+
+@pytest.mark.tier(1)
+def test_osp_vmware_67_test_vm_name_with_punycode_characters():
+    """
+    title: OSP: vmware 67- Test VM name with Punycode characters
+    Polarion:
+        assignee: sshveta
+        initialEstimate: 1/4h
+        caseimportance: medium
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.10
+        casecomponent: V2V
+        title: Test customize request security group
+        testSteps:
+            1. Create infrastructure mapping for vmware67 to OSP
+            2. Create migration plan with infra map and VM with punycode chars
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. Successful migration from vmware to OSP
+    """
+    pass
+
+
+@pytest.mark.tier(1)
+def test_osp_kill_the_v2v_process_in_the_middle_restart_evmserverd_should_resume_migration_pos():
```

Comment:
You can remove this too since https://github.com/ManageIQ/integration_tests/pull/7875 can add this anyways. Please note we are not supporting system restart and only supporting evmserver restart as you mentioned.